src/nixpacks/builder/docker/dockerfile_generation.rs

```diff
@@ -361,12 +361,19 @@ impl DockerfileGenerator for StartPhase {
                 start_cmd=start_cmd,}
             }
             None => {
+                // Copy over app files if provided, otherwise copy all files
+                let copy_cmds = match &self.only_include_files {
```

Comment:
START PHASE: if `only_include_files` is specified, then use it. Otherwise default to `COPY . /app`